doc/ExpertOptions.md

```diff
@@ -1441,13 +1441,23 @@ For an in depth discussion, please see [Missing Call Arguments](MissingCallArgum
 
 # Manifests
 
-In extreme cases, running compiled workflows can fail due to DNAnexus platform limits on the total size of the input and output JSON documents of a job. An example is a task with many inputs/outputs that is called in scatter over a large collection. In such a case, you can enable manifest support at compile time with the `-useManifests` option. This option causes each generated applet or workflow to accept inputs as a manifest, and to produce outputs as a manifest.
-
-A manifest is a JSON document that contains all the inputs/outputs that would otherwise be passed directly to/from the applet. A manifest can be specified in one of two ways: via a JSON input, or via a File input (where the file must exist on the platform).
+In extreme cases, running compiled workflows can fail due to DNAnexus platform limits on the total size of the input and 
+output JSON documents of a job. An example is a task with many inputs/outputs that is called in scatter over a large collection. 
+In such a case, you can enable manifest support at compile time with the `-useManifests` option. 
+This option causes each generated applet or workflow to accept inputs as a manifest, and to produce outputs as a manifest.
+
+A manifest is a JSON document that contains all the inputs/outputs that would otherwise be passed directly to/from the 
+workflow stage. A manifest can be specified in one of two ways: 
+1. A `.json` input file (see `Manifest JSON`). Here, when the workflow is **compiled** with `-inputs mymanifest.json`, a new 
```

Comment:
In this case invoking
```suggestion
1. A `.json` input file (see [Manifest JSON](#manifest-json)) is the recommended way to provide inputs in the manifest format.  `java -jar dxCompiler.jar -inputs mymanifest.json` will produce `mymanifest.dx.json` that can be passed to `dx run`.
```

Reply:
For my education, 1 ends up creating a json file with a  `input_manifest___` field and 2 is passed as input_manifest_files___ argument?

Reply:
I like your version more.
Re: question - yes this is for the [situation #1](https://github.com/dnanexus/dxCompiler/blob/34af2d06912bb083b83ecf159a95c62e9e91a92f/doc/ExpertOptions.md?plain=1#L1511)
This is for the [situation #2](https://github.com/dnanexus/dxCompiler/blob/34af2d06912bb083b83ecf159a95c62e9e91a92f/doc/ExpertOptions.md?plain=1#L1540)
Both are in the text